test.lua

```diff
@@ -120,6 +128,8 @@ do local _ENV = idl
 
 	func.getSupportedRenderers
 		"uint8_t"
+		.max  "uint8_t"
+		.enum "uint8_t" -- BUG: can't do :: in return types yet RendererType::Enum *"
```

Comment:
Why BUG ? 

I saw the enum type cast directly in bgfx 

https://github.com/bkaradzic/bgfx/blob/master/src/bgfx.cpp#L5381-L5384

Reply:
In earlier version there was issue putting "RendererType *" there, but since it switched to { "enum" } it works now.